src/buildercore/cfngen.py

```diff
@@ -385,18 +386,25 @@ def build_context_rds(pdata, context, existing_context):
         # may be present but None
         rds_password = generated_password
 
-    # TODO: shift the below under a 'rds' key
+    context['rds'] = pdata['aws']['rds']
+    context['rds'].update({
+        'deletion-policy': deletion_policy,
+    })
+
+    auto_rds_dbname = slugify(stackname, separator="") # lax--prod => laxprod
+    existing_rds_dbname = existing_context.get('rds_dbname')
+    override = lookup(pdata, 'aws.rds.db-name', None)
+    rds_dbname = override or existing_rds_dbname or auto_rds_dbname
```

Comment:
I think `existing` should come first ...

Reply:
previously we've been relying on the consistent behaviour of the slugify function to always slugify the given value the exact same way. Nothing wrong with that, but if we have access to the previously slugified and in-use dbname, we should prefer that.

Reply:
test

Reply:
> I think existing should come first ...

no, `override` is explicit, `existing` is implicit and `auto` is even more implicit.

Reply:
move this whole dbname wrangling into `trop.py`